main.cpp

```diff
@@ -57,9 +61,49 @@ void variableDeclarations()
     //example:
     int number = 2; //declaration of a primitive named 'number' with an initial value of '2'
     
-    
-    
     ignoreUnused(number); //passing each variable declared to the ignoreUnused() function
+
+    // char
+    char studentGrade = 'B';
+    char middleInitial = 'T';
+    char lowercaseLetter = 'g';
+
+    // int
+    int customerAge = 21;
+    int speedLimit = 55;
+    int outsideTemp = -10;
+
+    // float 
+    float studentGPA = 3.5f;
+    float interestRate = 12.5f;
+    float percentComplete = 58.f;
+
+    // double
+    double furnaceTemp = 1800.0;
+    double discountAmount = -8345.76;
+    double lotteryJackpot = 308000000.2;
+
+    // bool
+    bool accountInArrears = false;
+    bool collegeDegree = true;
+    bool roomOccupied = false;
+
+    // calls
+    ignoreUnused(studentGrade);
+    ignoreUnused(middleInitial);
+    ignoreUnused(lowercaseLetter);
+    ignoreUnused(customerAge);
+    ignoreUnused(speedLimit);
+    ignoreUnused(outsideTemp);
+    ignoreUnused(studentGPA);
+    ignoreUnused(interestRate);
+    ignoreUnused(percentComplete);
+    ignoreUnused(furnaceTemp);
+    ignoreUnused(discountAmount);
+    ignoreUnused(lotteryJackpot);
+    ignoreUnused(accountInArrears);
+    ignoreUnused(collegeDegree);
+    ignoreUnused(roomOccupied);
```

Comment:
Use a single call of `ignoreUnused`, like you're doing in the functions below.
- [ ] revise

Reply:
resvision completed